lib/v2/bilibili/cache.js

```diff
@@ -1,12 +1,22 @@
 const got = require('@/utils/got');
 const utils = require('./utils');
+const { CookieJar} = require('tough-cookie');
 
 module.exports = {
     getCookie: (ctx) => {
         const key = 'bili-cookie';
         return ctx.cache.tryGet(key, async () => {
-            const response = await got('https://www.bilibili.com/');
-            return response.headers['set-cookie'];
+            // default Referer: https://www.bilibili.com is limited
+            // Bilibili return cookies with multiple set-cookie
+            const cookieJar = new CookieJar();
+            const url = 'https://www.bilibili.com/';
+            await got('https://www.bilibili.com/', {
+                headers: {
+                    Referer: 'https://www.bilibili.com/'
+                },
+                cookieJar
+            });
+            return cookieJar.getCookieStringSync(url);
```

Comment:
```suggestion
            const url = 'https://www.bilibili.com/';
            const response = await got(url, {
                headers: {
                    Referer: url,
                },
            });
            return response.headers['set-cookie'].map((cookie) => cookie.split(';')[0]).join('; ');
```
Since the cookie jar is not used in other got requests, you can use `map()`, `split()` and `join()` to achieve the same thing.